docs/getting-started-guides/windows/index.md

```diff
@@ -1,144 +1,262 @@
+- Improved support for pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per pod
+- Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
+- Container Runtime Interface (CRI) pod and node level statistics
+- Support for kubeadm commands to add Windows Server nodes to a Kubernetes environment
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.5.
+The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server 2016 or later
+
+**Note:** Windows Server Containers on Kubernetes is a Beta feature in Kubernetes v1.9
+
+## Build
+We recommend using the release binaries that can be found at [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases). Look for the Node Binaries section by visiting the binary downloads link. 
+
+If you wish to build the code yourself, please follow the next instructions:
+
+1. Install the pre-requisites on a Linux host:
+
+```
+sudo apt-get install curl git build-essential docker.io conntrack
+```
+
+2. Run the following commands to build kubelet and kube-proxy:
+
+```code
+K8SREPO="github.com/kubernetes/kubernetes"
+go get -d $K8SREPO
+# Note: the above command may spit out a message about 
+#       "no Go files in...", but it can be safely ignored!
+
+cd $GOPATH/src/k8s.io/kubernetes
+# Build the kubelet
+KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet
+
+# Build the kube-proxy
+KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy
+
+# You will find the output binaries under the folder _output/local/bin/windows/
+```
+
+More detailed build instructions will be maintained and kept up to date [here](https://github.com/microsoft/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries.md)
 
 ## Prerequisites
-In Kubernetes version 1.5, Windows Server Containers for Kubernetes is supported using the following:
+In Kubernetes version 1.9 or later, Windows Server Containers for Kubernetes are supported using the following:
 
-1. Kubernetes control plane running on existing Linux infrastructure (version 1.5 or later).
+1. Kubernetes control plane running on existing Linux infrastructure (version 1.9 or later).
 2. Kubenet network plugin setup on the Linux nodes.
-3. Windows Server 2016 (RTM version 10.0.14393 or later).
-4. Docker Version 1.12.2-cs2-ws-beta or later for Windows Server nodes (Linux nodes and Kubernetes control plane can run any Kubernetes supported Docker Version).
+3. Windows Server 2016 RTM or later. Windows Server version 1709 or later is preferred; unlocks key capabilities like shared network namespace.
```

Comment:
Minor nit: "is preferred; _**it**_ unlocks key...". Semicolons should always be replaceable by periods.

Reply:
fixed